### ISSUE TITLE
Mimic loss returns nan

### ISSUE THREAD

**Issue by sara782** [wozhouh/my-Mobile-Mask-RCNN#3]
Hello,
Your jupyter notebook train_on_coco shows a perfectly functional mimic loss, but when I try to run the file I get a nan value, and therefore the total loss is also a nan
I've seen a comment from you where it says "modify mimic loss to l2 loss but not work yet", is that in reference to the one coded, or one you wish to update in the future
@wozhouh @waleedka please, I'd really appreciate it if you'd reply. Or if you could direct me to another repository with distilled object detection, preferably with keras or tensorflow
